### PR TITLE
feat: moved the detail's to a dotenv file

### LINKED ISSUE #3: Backend: More secure email, password configuration
### 🚀Feature Request
Backend: More secure email, password configuration

### Description
Currently email and password are stored in __init__ file which is **_really_** insecure
Change the configuration such that it is more secure

### Describe the solution you'd like
Hint: Research environment variables and how to use them in python (FastAPI)

### PR DESCRIPTION
**Pull request by lokesh-wagh**
## Description

I have moved all the sensitive details to a .env folder

then those variable's are being loaded using load__dotenv method 

this required me to install python-dotenv package
I am using os.enviorn.get to get the variable's by passing appropriate key's

## Related Issue
#3 

Fixes # (issue)
fixes #3 
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist:

- [x] I have registered myself at [Contrihub website](https://sac.mnnit.ac.in/contrihub).
- [x] My code follows the code style of this project.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

### CODE REVIEW

File: frontend/package.json
Comment (PriyavKaneria) [inline frontend/package.json:1]: Animate on scroll library added unnecessarily

File: api/app/api.py
Comment (PriyavKaneria) [inline api/app/api.py:1]: ENV variable names can be more user friendly

File: api/app/__pycache__/api.cpython-312.pyc
Comment (PriyavKaneria) [inline api/app/__pycache__/api.cpython-312.pyc:1]: Pycache should be added to gitignore

File: .idea/vcs.xml
Comment (PriyavKaneria) [inline .idea/vcs.xml:1]: Unnecessary IDE folder .idea has been pushed

**Review by PriyavKaneria — COMMENTED** [review]
I will close the PR due to non-assignment, but I have still reviewed it for improvement. You may or may not make the changes. Keep the points in mind for PR of any other assigned issue

**Comment by lokesh-wagh** [comment]
Thank you for the insight's sir

**Pull request closed without merging**